examples/incremental_learning/helmet_detection_incremental_train/README.md

```diff
@@ -56,11 +60,10 @@ apiVersion: sedna.io/v1alpha1
 kind: Dataset
 metadata:
   name: incremental-dataset
-  namespace: sedna-test
 spec:
   url: "/data/helmet_detection/train_data/train_data.txt"
   format: "txt"
-  nodeName: "cloud0"
+  nodeName: $WORKER_NODE
```

Comment:
where is the definition of $WORKER_NODE?

Reply:
ok， i will definit it in the foreword